package.json

```diff
@@ -161,14 +170,13 @@
     "faker": "^4.1.0",
     "friendly-errors-webpack-plugin": "^1.7.0",
     "graphql-tools": "^3.0.2",
-    "html5": "*",
```

Comment:
This was leading to `node-gyp` errors -- i looked around the codebase and couldn't find anything related, so just removed.

Reply:
Ah, I tried to track that down a while ago, removing a lot of dependencies and never caught it. Thanks.

Reply:
The magic incantation is `yarn why <package-name>`